### DISCUSSION TITLE
Slicer ?

### DISCUSSION THREAD

**Discussion by toboul70** [PHPOffice/PhpSpreadsheet#2440] [Q&A] [answered]
Hello

How to put a pattern slicer on a cell ?

Th
![example](https://user-images.githubusercontent.com/34251846/145196722-524c9485-7bfc-40ff-b4b3-dc6dc86517f7.png)
x

**Comment by oleibman** [comment, marked as answer]
There are a number of FILL_PATTERN constants in Style/Fill.php. It looks like FILL_PATTERN_DARKUP matches your request.
```php
$sheet->getStyle('G3')->getFill()->setFillType(Fill::FILL_PATTERN_DARKUP);
```

**Reply by toboul70** [reply]
Yes ! Great.

Many Thx.
